policyengine_us/variables/household/expense/charitable/foster_care_charitable_organization.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class foster_care_charitable_organization(Variable):
```

Comment:
Make this a float.

This needs to be in the Arizona folder because of the bolded:
>5. " Qualifying foster care charitable organization" means a qualifying charitable organization that is exempt from federal income taxation under section 501(c)(3) of the internal revenue code or is a designated community action agency that receives community services block grant program monies pursuant to 42 United States Code section 9901 and that each operating year provides services to **at least two hundred qualified individuals in this state and spends at least fifty percent of its budget on services to qualified individuals in this state.**
```suggestion
class az_charitable_contributions_to_qualifying_foster_care_organizations(Variable):
```